humanguide/index.rst

```diff
@@ -1,7 +1,58 @@
 Human interface guidelines
 ==========================
 
-Ubuntu Touch follows a clean, beautiful, modern and (above all) convergent design. We aim to keep developing the design started by Canonical.
+Ubuntu Touch follows a modern and convergent design philosophy called Suru.
+We aim to keep developing the design started by Canonical.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubports we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translated into our design philosophy called Suru.
+
+- Suru items from the UTouch values alluding to Japanese culture.
```

Comment:
I think that must be a historical Canonical/Ubuntu reference and I think there it is best to keep the original formulation, so "from the UTouch values" -> "from the Ubuntu values"